ibis/util.py

```diff
@@ -101,7 +102,11 @@ def coerce_to_dataframe(data: Any, names: List[str]) -> pd.DataFrame:
         series = [data.apply(lambda t: t[i]) for i in range(num_cols)]
         result = pd.concat(series, axis=1)
     elif isinstance(data, (tuple, list)):
-        result = pd.concat(data, axis=1)
+        if isinstance(data[0], pd.Series):
+            result = pd.concat(data, axis=1)
+        else:
+            # Promote scalar to Series
+            result = pd.concat([pd.Series([v]) for v in data], axis=1)
```

Comment:
does this have the right name? IIUC this is a scalar returned for an aggregation. but why does this hit here?

Reply:
The names are set below inline 116.

This is used to return multiple fields (destruct) for an aggregation. In which case, user will return sth like 

```
return 1.0, 2.0
```

from the UDF and therefore we coerce the output to the proper shape in order for it to work with in the logic in:

https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/execution/generic.py#L436

Reply:
Also, it is not inefficient to do so because this is only hit when doing aggregation without groupby and therefore, this is done only once for the entire aggregation node.

Reply:
ok this sounds fine